app/views/collection/show.html.slim

```diff
@@ -138,3 +138,13 @@
     =deeds_for({ :collection => @collection, :limit => 20, :types => DeedType.collection_edits, :long_view => true })
 
 =render({ :partial => '/shared/collection_footer' })
+
+=link_to(collection_new_mobile_user_path(@collection.id), :data => { litebox: { hash: 'new-mobile-user' }}, id: 'new_mobile_user_link', style: "display: none")
+  span =""
+
+-if @new_mobile_user
+  -content_for :javascript
+    javascript: 
+      $(document).ready(function(){
+        $('#new_mobile_user_link').trigger('click');
+      });
```

Comment:
This is how I got the popup to show up only when `@new_mobile_user` is true, but this code seems weird & bad practice to me. Do y'all have any insight into a way to do this better?